package.json

```diff
@@ -88,6 +73,7 @@
     "express-bearer-token": "^2.4.0",
     "express-cache-on-demand": "^1.0.3",
     "express-session": "^1.17.1",
+    "floating-vue": "^2.0.0-beta.24",
```

Comment:
replaces v-tooltip with support for vue 3. Api has quite changed so need refacto, mainly in `AposContextMenu`